conf/messages

```diff
@@ -14,13 +14,14 @@ common.account = account
 common.aDifferentAmount = A different amount
 common.back = Back
 common.benefits = Benefits
+common.employmentBenefits = Employment benefits
 common.change = Change
 common.checkWithEmployer.agent = Check with your client’s employer if you are unsure.
 common.checkWithEmployer.individual = Check with your employer if you are unsure.
 common.continue = Continue
 common.currency.hint = For example, £600 or £193.54
 common.employmentDetails = Employment details
-common.employmentDetailsAndBenefits = Employer information
+common.employerInformation = Employer information
```

Comment:
This does not look like a common text. It should probably go to its dedicated page location.

Reply:
sure

Reply:
done